src/Security/test/AuthSamples.FunctionalTests/AuthSamples.FunctionalTests.csproj

```diff
@@ -46,34 +51,44 @@
     </ItemGroup>
     <Copy
       SourceFiles="@(_PublishFiles)"
-      DestinationFolder="$(PublishDir)" />
+      DestinationFolder="$(PublishDir)"
+      ContinueOnError="true" />
```

Comment:
Same question about failing fast.

Reply:
I can back these out for now, I'm not sure why these projects are failing with copy file exists errors on the arm64 builds, these were trying to work around those errors, they are skipped for now on arm64